libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -225,6 +228,32 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
 #endif
 #endif  // HAL_BARO_WIND_COMP_ENABLED
 
+#if AP_BARO_THST_COMP_ENABLED
+    // @Param: 1_THST_SCALE
+    // @DisplayName: Thrust compensation
+    // @Description: User provided thrust scaling in Pascals. This is used to adjust linearly based on the thrust output for local pressure difference induced by the props.
+    // @Range: -300 300
+    // @User: Advanced
+    AP_GROUPINFO("1_THST_SCALE", 25, AP_Baro, sensors[0].mot_scale, 0),
```

Comment:
I find it confusing that index 25 is here. Why not at the end? If you want to keep them here please add a comment at the end of the table noting that 25,26 and 27 are used here.

Reply:
Fixed